src/table/components/TableWrapper.jsx

```diff
@@ -179,24 +200,14 @@ export default function TableWrapper(props) {
       </TableContainer>
       <Paper sx={paperTablePaginationStyle}>
         <TablePagination
-          sx={constraints.active && { display: 'none' }}
+          sx={tablePaginationStyle}
```

Comment:
As we decided to simplify the component, otherwise the return () section will become super long
<img width="923" alt="Screenshot 2022-03-01 at 19 15 00" src="https://user-images.githubusercontent.com/4471489/156225646-b821e2dd-f3d7-403c-b875-704f43dc1dbf.png">